server/package-lock.json

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
Should this file exist? It may be a result of running `npm install` in the `server/` directory instead of `yarn`? If so, lets take it out.